Lumberjack GUI not opening

### Minecolonies version

version 1.10.2-0.8.2817

### Expected behavior

Buildable Hut and openable GUI for Lumberjack

### Actual behaviour

[12:12:57] [Server thread/ERROR] [minecolonies]: Unknown Building type 'com.minecolonies.coremod.tileentities.TileEntityColonyBuilding' or missing constructor of proper format.
java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_121]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.8.0_121]
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.8.0_121]
        at java.lang.reflect.Constructor.newInstance(Unknown Source) ~[?:1.8.0_121]
        at com.minecolonies.coremod.colony.buildings.AbstractBuilding.create(AbstractBuilding.java:382) ~[AbstractBuilding.class:?]
        at com.minecolonies.coremod.colony.Colony.addNewBuilding(Colony.java:1605) ~[Colony.class:?]
        at com.minecolonies.coremod.blocks.AbstractBlockHut.func_180633_a(AbstractBlockHut.java:262) ~[AbstractBlockHut.class:?]
        at com.minecolonies.coremod.blocks.AbstractBlockHut.onBlockPlacedByBuildTool(AbstractBlockHut.java:291) ~[AbstractBlockHut.class:?]
        at com.minecolonies.coremod.network.messages.BuildToolPlaceMessage.handleHut(BuildToolPlaceMessage.java:174) ~[BuildToolPlaceMessage.class:?]
        at com.minecolonies.coremod.network.messages.BuildToolPlaceMessage.messageOnServerThread(BuildToolPlaceMessage.java:133) ~[BuildToolPlaceMessage.class:?]
        at com.minecolonies.coremod.network.messages.BuildToolPlaceMessage.messageOnServerThread(BuildToolPlaceMessage.java:36) ~[BuildToolPlaceMessage.class:?]
        at com.minecolonies.coremod.network.messages.AbstractMessage.lambda$onMessage$0(AbstractMessage.java:24) ~[AbstractMessage.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_121]
        at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_121]
        at net.minecraft.util.Util.func_181617_a(SourceFile:45) [h.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:668) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387) [ld.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_121]
Caused by: java.lang.NoSuchMethodError: net.minecraft.item.Item.func_150895_a(Lnet/minecraft/item/Item;Lnet/minecraft/creativetab/CreativeTabs;Ljava/util/List;)V
        at com.minecolonies.coremod.colony.buildings.BuildingLumberjack.calcSaplings(BuildingLumberjack.java:129) ~[BuildingLumberjack.class:?]
        at com.minecolonies.coremod.colony.buildings.BuildingLumberjack.<init>(BuildingLumberjack.java:91) ~[BuildingLumberjack.class:?]
        ... 20 more
[12:12:57] [Server thread/ERROR] [minecolonies]: Colony 1 unable to create AbstractBuilding for class com.minecolonies.coremod.blocks.BlockHutLumberjack at BlockPos{x=365, y=67, z=238}
[12:12:57] [Server thread/ERROR] [minecolonies]: BuildTool: building is null!

### Steps to reproduce the problem

1. set block (with wand)
2. try to open
3. fail
4. try not to cry
5. cry alot

--- a/src/api/java/com/minecolonies/api/util/constant/TranslationConstants.java
+++ b/src/api/java/com/minecolonies/api/util/constant/TranslationConstants.java
@@ -102,7 +102,9 @@ public final class TranslationConstants
     @NonNls
     public static final String COM_MINECOLONIES_COREMOD_GUI_WORKERHUTS_RETRIEVE_OFF           = "com.minecolonies.coremod.gui.workerHuts.retrieveOff";
     @NonNls
-    public static final String COM_MINECOLONIES_COREMOD_GUI_WORKERHUTS_LEVEL_0                 = "com.minecolonies.coremod.gui.workerHuts.level0";
+    public static final String COM_MINECOLONIES_COREMOD_GUI_WORKERHUTS_LEVEL_0                = "com.minecolonies.coremod.gui.workerHuts.level0";
+    @NonNls
+    public static final String COM_MINECOLONIES_COREMOD_GUI_LUMBERJACK                        = "com.minecolonies.coremod.job.Lumberjack";
 
     private TranslationConstants()
     {

--- a/src/main/java/com/minecolonies/coremod/blocks/BlockHutLumberjack.java
+++ b/src/main/java/com/minecolonies/coremod/blocks/BlockHutLumberjack.java
@@ -3,7 +3,7 @@ package com.minecolonies.coremod.blocks;
 import org.jetbrains.annotations.NotNull;
 
 /**
- * Hut for the citizen.
+ * Hut for the lumberjack.
  * No different from {@link AbstractBlockHut}
  */
 public class BlockHutLumberjack extends AbstractBlockHut

--- a/src/main/java/com/minecolonies/coremod/client/gui/WindowHutLumberjack.java
+++ b/src/main/java/com/minecolonies/coremod/client/gui/WindowHutLumberjack.java
@@ -1,8 +1,8 @@
 package com.minecolonies.coremod.client.gui;
 
+import com.minecolonies.coremod.entity.ai.item.handling.ItemStorage;
 import com.minecolonies.api.util.LanguageHandler;
 import com.minecolonies.api.util.constant.Constants;
-import com.minecolonies.blockout.Log;
 import com.minecolonies.blockout.Pane;
 import com.minecolonies.blockout.controls.Button;
 import com.minecolonies.blockout.controls.ItemIcon;
@@ -18,9 +18,7 @@ import org.jetbrains.annotations.NotNull;
 import java.util.LinkedHashMap;
 import java.util.Map;
 
-import static com.minecolonies.api.util.constant.TranslationConstants.COM_MINECOLONIES_COREMOD_GUI_BAKER;
-import static com.minecolonies.api.util.constant.TranslationConstants.COM_MINECOLONIES_COREMOD_GUI_WORKERHUTS_RETRIEVE_OFF;
-import static com.minecolonies.api.util.constant.TranslationConstants.COM_MINECOLONIES_COREMOD_GUI_WORKERHUTS_RETRIEVE_ON;
+import static com.minecolonies.api.util.constant.TranslationConstants.*;
 
 /**
  * Window for the fisherman hut.
@@ -55,7 +53,7 @@ public class WindowHutLumberjack extends AbstractWindowWorkerBuilding<BuildingLu
     /**
      * Id of the pages view.
      */
-    private static final String VIEW_PAGES                = "pages";
+    private static final String VIEW_PAGES                           = "pages";
 
     /**
      * Scrolling list containing the saplings.
@@ -65,7 +63,7 @@ public class WindowHutLumberjack extends AbstractWindowWorkerBuilding<BuildingLu
     /**
      * List of saplings the lumberjack should, or should not fell (true if should, false if should not).
      */
-    private final Map<ItemStack, Boolean> treesToFell = new LinkedHashMap<>();
+    private final Map<ItemStorage, Boolean> treesToFell = new LinkedHashMap<>();
 
     /**
      * The building of the lumberjack (Client side representation).
@@ -113,13 +111,13 @@ public class WindowHutLumberjack extends AbstractWindowWorkerBuilding<BuildingLu
             @Override
             public void updateElement(final int index, @NotNull final Pane rowPane)
             {
-                final ItemStack sapling = treesToFell.keySet().toArray(new ItemStack[treesToFell.size()])[index];
+                final ItemStack sapling = treesToFell.keySet().toArray(new ItemStorage[treesToFell.size()])[index].getItemStack();
                 rowPane.findPaneOfTypeByID("symbol", ItemIcon.class).setItem(sapling);
                 rowPane.findPaneOfTypeByID("name", Label.class).setLabelText(sapling.getDisplayName());
 
                 final Button switchButton = rowPane.findPaneOfTypeByID(BUTTON_CURRENT_SAPLING, Button.class);
 
-                if(treesToFell.get(sapling))
+                if(treesToFell.get(new ItemStorage(sapling)))
                 {
                     switchButton.setLabel(ON);
                 }
@@ -138,11 +136,11 @@ public class WindowHutLumberjack extends AbstractWindowWorkerBuilding<BuildingLu
         {
             final int row = saplingsList.getListElementIndexByPane(button);
 
-            final ItemStack saplingStack = treesToFell.keySet().toArray(new ItemStack[treesToFell.size()])[row];
-            Log.getLogger().warn("Sapling: " + saplingStack.getItem().getUnlocalizedName() + " to: " + !treesToFell.get(saplingStack));
+            final ItemStorage saplingStack = treesToFell.keySet().toArray(new ItemStorage[treesToFell.size()])[row];
+
             final boolean shouldCut = !treesToFell.get(saplingStack);
             treesToFell.put(saplingStack, shouldCut);
-            MineColonies.getNetwork().sendToServer(new LumberjackSaplingSelectorMessage(building, saplingStack, shouldCut));
+            MineColonies.getNetwork().sendToServer(new LumberjackSaplingSelectorMessage(building, saplingStack.getItemStack(), shouldCut));
 
             this.ownBuilding.treesToFell.clear();
             this.ownBuilding.treesToFell.putAll(treesToFell);
@@ -177,7 +175,7 @@ public class WindowHutLumberjack extends AbstractWindowWorkerBuilding<BuildingLu
     @Override
     public String getBuildingName()
     {
-        return COM_MINECOLONIES_COREMOD_GUI_BAKER;
+        return COM_MINECOLONIES_COREMOD_GUI_LUMBERJACK;
     }
 }
 

--- a/src/main/java/com/minecolonies/coremod/client/gui/WindowWareHouseBuilding.java
+++ b/src/main/java/com/minecolonies/coremod/client/gui/WindowWareHouseBuilding.java
@@ -9,7 +9,7 @@ import org.jetbrains.annotations.NotNull;
  */
 public class WindowWareHouseBuilding extends AbstractWindowBuilding<BuildingWareHouse.View>
 {
-    private static final String HOME_BUILDING_RESOURCE_SUFFIX = ":gui/windowHutHome.xml";
+    private static final String HOME_BUILDING_RESOURCE_SUFFIX = ":gui/windowHutWarehouse.xml";
 
     /**
      * Creates the Window object.

--- a/src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java
+++ b/src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java
@@ -2,7 +2,6 @@ package com.minecolonies.coremod.colony.buildings;
 
 import com.minecolonies.api.util.constant.ToolType;
 import com.minecolonies.api.util.ItemStackUtils;
-import com.minecolonies.blockout.Log;
 import com.minecolonies.blockout.views.Window;
 import com.minecolonies.coremod.MineColonies;
 import com.minecolonies.coremod.achievements.ModAchievements;

--- a/src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java
+++ b/src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java
@@ -2,6 +2,7 @@ package com.minecolonies.coremod.colony.buildings;
 
 import com.minecolonies.api.util.constant.ToolType;
 import com.minecolonies.api.util.ItemStackUtils;
+import com.minecolonies.blockout.Log;
 import com.minecolonies.blockout.views.Window;
 import com.minecolonies.coremod.achievements.ModAchievements;
 import com.minecolonies.coremod.client.gui.WindowHutLumberjack;
@@ -86,12 +87,16 @@ public class BuildingLumberjack extends AbstractBuildingWorker
             saplings.addAll(OreDictionary.getOres(OreDictionary.getOreName(i)));
         }
 
-        if (treesToFell.isEmpty())
+        if(c.getWorld().isRemote)
         {
-            treesToFell.putAll(calcSaplings(saplings));
-        }
+            if (treesToFell.isEmpty())
+            {
+                Log.getLogger().warn("Try to do saplings");
+                treesToFell.putAll(View.calcSaplings(saplings));
+            }
 
-        markDirty();
+            markDirty();
+        }
     }
 
     /**
@@ -102,10 +107,7 @@ public class BuildingLumberjack extends AbstractBuildingWorker
      */
     public void setTreeToCut(final ItemStack stack, final boolean cut)
     {
-        if (treesToFell.containsKey(new ItemStorage(stack)))
-        {
-            treesToFell.put(new ItemStorage(stack), cut);
-        }
+        treesToFell.put(new ItemStorage(stack), cut);
     }
 
     /**
@@ -118,38 +120,22 @@ public class BuildingLumberjack extends AbstractBuildingWorker
         return Collections.unmodifiableMap(treesToFell);
     }
 
-    private static Map<ItemStorage, Boolean> calcSaplings(final List<ItemStack> saplings)
-    {
-        final Map<ItemStorage, Boolean> finalSaplings = new LinkedHashMap<>();
-        for (final ItemStack saps : saplings)
-        {
-            if (saps.getHasSubtypes())
-            {
-                final List<ItemStack> list = new ArrayList<>();
-                saps.getItem().getSubItems(saps.getItem(), null, list);
-
-                for (final ItemStack stack : list)
-                {
-                    finalSaplings.put(new ItemStorage(stack), true);
-                }
-            }
-        }
-        return finalSaplings;
-    }
-
     @Override
     public void readFromNBT(@NotNull final NBTTagCompound compound)
     {
-        super.readFromNBT(compound);
-        treesToFell.clear();
-
-        final NBTTagList saplingTagList = compound.getTagList(TAG_SAPLINGS, Constants.NBT.TAG_COMPOUND);
-        for (int i = 0; i < saplingTagList.tagCount(); ++i)
+        if(treesToFell.isEmpty())
         {
-            final NBTTagCompound saplingCompound = saplingTagList.getCompoundTagAt(i);
-            final ItemStack stack = ItemStack.loadItemStackFromNBT(saplingCompound);
-            final boolean cut = saplingCompound.getBoolean(TAG_CUT);
-            treesToFell.put(new ItemStorage(stack), cut);
+            super.readFromNBT(compound);
+            treesToFell.clear();
+
+            final NBTTagList saplingTagList = compound.getTagList(TAG_SAPLINGS, Constants.NBT.TAG_COMPOUND);
+            for (int i = 0; i < saplingTagList.tagCount(); ++i)
+            {
+                final NBTTagCompound saplingCompound = saplingTagList.getCompoundTagAt(i);
+                final ItemStack stack = ItemStack.loadItemStackFromNBT(saplingCompound);
+                final boolean cut = saplingCompound.getBoolean(TAG_CUT);
+                treesToFell.put(new ItemStorage(stack), cut);
+            }
         }
     }
 
@@ -281,7 +267,7 @@ public class BuildingLumberjack extends AbstractBuildingWorker
         /**
          * List of saplings the lumberjack should, or should not fell (true if should, false if should not).
          */
-        public final Map<ItemStack, Boolean> treesToFell = new LinkedHashMap<>();
+        public final Map<ItemStorage, Boolean> treesToFell = new LinkedHashMap<>();
 
         /**
          * Public constructor of the view, creates an instance of it.
@@ -303,7 +289,19 @@ public class BuildingLumberjack extends AbstractBuildingWorker
             {
                 final ItemStack stack = ByteBufUtils.readItemStack(buf);
                 final boolean cut = buf.readBoolean();
-                treesToFell.put(stack, cut);
+                treesToFell.put(new ItemStorage(stack), cut);
+            }
+
+            if(treesToFell.isEmpty())
+            {
+                final List<ItemStack> saplings = new ArrayList<>();
+                final int[] saplingId = OreDictionary.getOreIDs(new ItemStack(Blocks.SAPLING));
+
+                for (final int i : saplingId)
+                {
+                    saplings.addAll(OreDictionary.getOres(OreDictionary.getOreName(i)));
+                }
+                treesToFell.putAll(calcSaplings(saplings));
             }
         }
 
@@ -326,6 +324,30 @@ public class BuildingLumberjack extends AbstractBuildingWorker
         public Skill getSecondarySkill()
         {
             return Skill.CHARISMA;
+        }
+
+        /**
+         * Calculates all saplings ingame and return an itemStorage map of it.
+         * @param saplings the saplings.
+         * @return the itemStorage map.
+         */
+        public static Map<ItemStorage, Boolean> calcSaplings(final List<ItemStack> saplings)
+        {
+            final Map<ItemStorage, Boolean> finalSaplings = new LinkedHashMap<>();
+            for (final ItemStack saps : saplings)
+            {
+                if (saps.getHasSubtypes())
+                {
+                    final List<ItemStack> list = new ArrayList<>();
+                    saps.getItem().getSubItems(saps.getItem(), null, list);
+
+                    for (final ItemStack stack : list)
+                    {
+                        finalSaplings.put(new ItemStorage(stack), true);
+                    }
+                }
+            }
+            return finalSaplings;
         }
     }
 }

--- a/src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java
+++ b/src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java
@@ -4,6 +4,7 @@ import com.minecolonies.api.util.constant.ToolType;
 import com.minecolonies.api.util.ItemStackUtils;
 import com.minecolonies.blockout.Log;
 import com.minecolonies.blockout.views.Window;
+import com.minecolonies.coremod.MineColonies;
 import com.minecolonies.coremod.achievements.ModAchievements;
 import com.minecolonies.coremod.client.gui.WindowHutLumberjack;
 import com.minecolonies.coremod.colony.CitizenData;
@@ -12,6 +13,7 @@ import com.minecolonies.coremod.colony.ColonyView;
 import com.minecolonies.coremod.colony.jobs.AbstractJob;
 import com.minecolonies.coremod.colony.jobs.JobLumberjack;
 import com.minecolonies.coremod.entity.ai.item.handling.ItemStorage;
+import com.minecolonies.coremod.network.messages.LumberjackSaplingSelectorMessage;
 import io.netty.buffer.ByteBuf;
 import net.minecraft.init.Blocks;
 import net.minecraft.item.ItemStack;
@@ -302,6 +304,11 @@ public class BuildingLumberjack extends AbstractBuildingWorker
                     saplings.addAll(OreDictionary.getOres(OreDictionary.getOreName(i)));
                 }
                 treesToFell.putAll(calcSaplings(saplings));
+
+                for(final Map.Entry<ItemStorage, Boolean> entry : treesToFell.entrySet())
+                {
+                    MineColonies.getNetwork().sendToServer(new LumberjackSaplingSelectorMessage(this, entry.getKey().getItemStack(), entry.getValue()));
+                }
             }
         }
 

--- a/src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java
+++ b/src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java
@@ -81,24 +81,6 @@ public class BuildingLumberjack extends AbstractBuildingWorker
 
         final ItemStack stack = new ItemStack(Blocks.SAPLING);
         keepX.put(new ItemStorage(stack, false), SAPLINGS_TO_KEEP);
-        final int[] saplingId = OreDictionary.getOreIDs(new ItemStack(Blocks.SAPLING));
-
-        final List<ItemStack> saplings = new ArrayList<>();
-        for (final int i : saplingId)
-        {
-            saplings.addAll(OreDictionary.getOres(OreDictionary.getOreName(i)));
-        }
-
-        if(c.getWorld().isRemote)
-        {
-            if (treesToFell.isEmpty())
-            {
-                Log.getLogger().warn("Try to do saplings");
-                treesToFell.putAll(View.calcSaplings(saplings));
-            }
-
-            markDirty();
-        }
     }
 
     /**

--- a/src/main/java/com/minecolonies/coremod/commands/CommandEntryPoint.java
+++ b/src/main/java/com/minecolonies/coremod/commands/CommandEntryPoint.java
@@ -37,10 +37,10 @@ public class CommandEntryPoint extends CommandBase
     }
 
     /**
-     *
-     * @param server
-     * @param sender
-     * @return
+     * Check if the player has the permission to use commands.
+     * @param server the server to check for.
+     * @param sender the sender of the command.
+     * @return true if so.
      */
     @Override
     public boolean checkPermission(final MinecraftServer server, final ICommandSender sender)

--- a/src/main/resources/assets/minecolonies/gui/windowHutWarehouse.xml
+++ b/src/main/resources/assets/minecolonies/gui/windowHutWarehouse.xml
@@ -1,71 +1,9 @@
 <window xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         inherit="minecolonies:gui/windowBase.xml"
         xsi:noNamespaceSchemaLocation="file:../../../../java/com/minecolonies/blockout/blockOut.xsd">
-    <switch id="pages" size="100% 200px" pos="0 7">
-        <view id="pageActions" size="100% 100%">
-            <layout source="minecolonies:gui/layoutWorkerBuildingDefault.xml"/>
-        </view>
-        <view id="pageSettings" size="100% 100%" pos="0 1">
-
-            <!-- width = 171 -->
-            <!-- xl = 52, xr = 109 -->
-
-            <!-- Blacksmith -->
-            <view size="100% 40" pos="0 0">
-                <label size="100% 11" pos="0 0"
-                       label="$(com.minecolonies.coremod.gui.warehouse.toBlacksmith)" color="black" textalign="BOTTOM_MIDDLE"/>
-
-                <view size="100% 20" pos="0 16">
-                    <itemicon size="20 20" pos="33 2" item="gold_ingot"/>
-                    <button size="30 20" pos="52 0" id="blacksmithGold"/>
-                    <itemicon size="20 20" pos="90 2" item="diamond"/>
-                    <button size="30 20" pos="109 0" id="blacksmithDiamond"/>
-                </view>
-            </view>
-
-            <!-- Stonemason -->
-            <view size="100% 64" pos="0 41">
-                <label size="100% 11" pos="0 0"
-                       label="$(com.minecolonies.coremod.gui.warehouse.toStonemason)" color="black" textalign="BOTTOM_MIDDLE"/>
-
-                <view size="100% 20" pos="0 16">
-                    <itemicon size="20 20" pos="33 2" item="cobblestone"/>
-                    <button size="30 20" pos="52 0" id="stonemasonCobblestone"/>
-                    <itemicon size="20 20" pos="90 2" item="sand"/>
-                    <button size="30 20" pos="109 0" id="stonemasonSand"/>
-                </view>
-
-                <view size="100% 20" pos="0 40">
-                    <itemicon size="20 20" pos="33 2" item="netherrack"/>
-                    <button size="30 20" pos="52 0" id="stonemasonNetherrack"/>
-                    <itemicon size="20 20" pos="90 2" item="quartz"/>
-                    <button size="30 20" pos="109 0" id="stonemasonQuartz"/>
-                </view>
-            </view>
-
-            <!-- Guards -->
-            <view size="100% 40" pos="0 110">
-                <label size="100% 11" pos="0 0"
-                       label="$(com.minecolonies.coremod.gui.warehouse.toGuards)" color="black" textalign="BOTTOM_MIDDLE"/>
-
-                <view size="100% 20" pos="0 16">
-                    <itemicon size="20 20" pos="33 2" item="iron_chestplate"/>
-                    <button size="30 20" pos="52 0" id="guardArmor"/>
-                    <itemicon size="20 20" pos="90 2" item="iron_sword"/>
-                    <button size="30 20" pos="109 0" id="guardWeapon"/>
-                </view>
-            </view>
-
-            <view size="100% 35" pos="0 155">
-                <label size="100% 11" pos="0 0"
-                       label="$(com.minecolonies.coremod.gui.warehouse.visitCitizenChests)" color="black" textalign="BOTTOM_MIDDLE"/>
-                <button size="30 20" pos="70 12" id="citizenChests"/>
-            </view>
-        </view>
-    </switch>
-
-    <button id="prevPage" align="TOP_LEFT" size="20 20" pos="10 5" label="&lt;&lt;"/>
-    <button id="nextPage" align="TOP_RIGHT" size="20 20" pos="10 5" label="&gt;&gt;"/>
+    <view id="pageActions" size="100% 200px" pos="0 7">
+        <layout source="minecolonies:gui/layoutBuildingDefault.xml"/>
+    </view>
 
     <button id="inventory"
             align="BOTTOM_MIDDLE" size="116 20" pos="0 13"

